helm/tweek/templates/authoring.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: authoring
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: authoring
+    spec:
+      containers:
+      - name: authoring
+        image: soluto/tweek-authoring:1.0.0-rc2
```

Comment:
shouldn't images/tags be variables?

Reply:
I don't think so. I think that the specific chart version should include specific versions of tweek components. But we can discuss it.